web/src/containers/IncentivePayments.js

```diff
@@ -121,42 +67,35 @@ class IncentivePayments extends Component {
                     >
                       {name}
                     </td>
-                    {yearsWithColors.map(({ year, colorLight }) => (
-                      <Fragment key={year}>
-                        {QUARTERS.map(q => (
-                          <td key={q} headers={tdHdrs(year, q)}>
-                            <InputComponent
-                              name={`${id}-payments-${year}-q${q}`}
-                              label={`${id} payments for ${year}, quarter ${q}`}
-                              wrapperClass="m0"
-                              className="m0 input input-condensed mono right-align"
-                              value={data[id][year][q] || ''}
-                              onChange={this.handleChange(id, year, q)}
-                              hideLabel
-                            />
-                          </td>
-                        ))}
-                        <td
-                          className={`bold mono right-align align-middle ${colorLight}`}
-                          headers={tdHdrs(year, 'subtotal')}
-                        >
-                          {formatNumber(type, totals[id].byYear[year])}
+                    <Fragment key={year}>
+                      {QUARTERS.map(q => (
+                        <td key={q} headers={tdHdrs(year, q)}>
+                          <InputComponent
+                            name={`${id}-payments-${year}-q${q}`}
+                            label={`${id} payments for ${year}, quarter ${q}`}
+                            wrapperClass="m0"
+                            className="m0 input input-condensed mono right-align"
+                            value={data[id][year][q] || ''}
+                            onChange={this.handleChange(id, year, q)}
+                            hideLabel
+                          />
                         </td>
-                      </Fragment>
-                    ))}
-                    <td
-                      className="bold mono right-align align-middle bg-gray-light"
-                      headers={tdHdrs('total', 'grand')}
-                    >
-                      {formatNumber(type, totals[id].allYears)}
-                    </td>
+                      ))}
+                      <td
+                        className="bold mono right-align align-middle"
+                        headers={tdHdrs(year, 'subtotal')}
+                      >
+                        {formatNumber(totals[id].byYear[year])}
```

Comment:
```suggestion
                        {formatNumber(type, totals[id].byYear[year])}
```

The subtotals column is ending up always as `--` - `formatNumber` needs the number type as its first argument